docs/components/table-view.md

```diff
@@ -922,6 +1093,47 @@ Slot props:
 </KTableView>
 ```
 
+### row-expanded
```

Comment:
Is there an event that is emitted when a row is expanded, before the slot content is actually shown?

We likely would want to do some data fetching once a user clicks to expand a row, before the row is actually expanded

Reply:
Good point, implemented.